src/styles/globals.css

```diff
@@ -42,7 +42,7 @@ a:hover {
 }
 
 #near-wallet-selector-modal .open {
-  z-index: 1001;
+  z-index: 5000;
```

Comment:
Need to check is this doesn't break the turn on notifications prompt 🤔

Reply:
@shelegdmitriy Fair point. I'd imagine the sign in modal should always take priority over anything else since you have to click the `Sign In` button to initiate it. Then it would close once you've connected your wallet or canceled.

Are you thinking the notifications prompt needs to render on top of the sign in modal?